rag-and-text-to-sql/app.py

```
@@ -0,0 +1,264 @@
+import fitz  # PyMuPDF for PDF extraction
+import docx  # python-docx for Word extraction
```

Ruff (0.8.2)

`docx` imported but unused

Remove unused import: `docx`

(F401)